InventoryManagementSystem/Inventory.cs

```diff
@@ -0,0 +1,103 @@
+﻿
+
+using System;
+namespace InventoryManagementSystem
+{
+	public class Inventory
+	{
+		private List<Product> products = new List<Product>();
+		public Inventory()
+		{
+		
+		}
+
+		public void AddProduct(Product product)
+		{
+			products.Add(product);
+			Console.WriteLine("Product added successfully.");
+		}
+
+		public void ViewProducts()
+		{
+			if(products.Count() == 0)
```

Comment:
There is a more performant way to check the count of the product list instead of using **Count()** method, can you search about it and use it?